index.html

```diff
@@ -166,17 +166,18 @@ <h1 class="primary-text-change">About<br>Myself</h1>
             </ul>
         </div>
     </section>
-    <form id="contact-form" class="contact-form" action="https://formspree.io/f/xoqrojoe" method="post">
+    <form novalidate id="contact-form" class="contact-form" action="https://formspree.io/f/xoqrojoe" method="post">

```

Comment:
- [ ] [OPTIONAL] Please remove `novalidate` property from the form. I think it should validate for empty fields and some basic validations. You are requested to add extra custom validation for the email field.

Reply:
Thank you very much for the kind review. I will definitely remove the novalidate, I was having a challenge with the javascript validation getting overshadowed by the html5 one though, it seems to take precedence so I couldn't see my javascript error message. What can I do for both to show up together?